Assets/Scripts/Interactable/NPCInteractable.cs

```diff
@@ -1,10 +1,13 @@
+using System.Collections.Generic;
 using UnityEngine;
 
 namespace Interactable
 {
     public class NPCInteractable : MonoBehaviour, IInteractable
     {
-        [SerializeField] private string interactText;
+        [SerializeField] protected string npcID;
+        [SerializeField] protected List<string> interactText;
```

Comment:
i think u should not change interactText to a list
since this interactText is shown like below img
![image](https://github.com/user-attachments/assets/1f2fed4d-d28f-4900-8f65-a37db2f12bc5)

Reply:
fix